code/modules/mining/minebot.dm

```diff
@@ -33,7 +33,6 @@
 						  /obj/item/stack/ore/bananium, /obj/item/stack/ore/tranquillite, /obj/item/stack/ore/glass,
 						  /obj/item/stack/ore/titanium)
 	healable = FALSE
-	loot = list(/obj/effect/decal/cleanable/robot_debris)
```

Comment:
Intended?

Reply:
Yep, same change was in your original pr actually.